scene/main/node.cpp

```diff
@@ -2498,6 +2498,9 @@ Node *Node::_duplicate(int p_flags, HashMap<const Node *, Node *> *r_duplimap) c
 
 	for (List<const Node *>::Element *N = node_tree.front(); N; N = N->next()) {
 		Node *current_node = node->get_node(get_path_to(N->get()));
+		if (instantiated && node != current_node) {
```

Comment:
This should only be the case if the scene has editable children disabled. If enabled, you need to do property duplication in the children.

Reply:
This case was indeed missed before.